chirp/wxui/settingsedit.py

```diff
@@ -131,9 +131,11 @@ def _apply_settings(self):
                                 if setting[j].get_value() != val:
                                     # setting value modified or not accepted in
                                     # validate callback, propagate back to GUI
-                                    prop = self._propgrid.propgrid.GetProperty(
-                                        name)
-                                    prop.SetValue(setting[j].get_value())
+                                    prop = self._group_control \
+                                        .GetCurrentPage() \
```

Comment:
This should just be `page` above right? We're applying all the settings at once, which means current page is probably not right either.